assignment6/rearranging_cars.py

```diff
@@ -0,0 +1,71 @@
+
+class Movement(object):
+    """This class stores information about one movement,
+    i.e. a car moving from position *fro* to position *to*.
+    """
+
+    def __init__(self, car, fro, to):
+        self.car = car
+        self.fro = fro
+        self.to = to
+
+    def print_movement(self):
+        print("Move car %s from %s to %s" %(self.car, self.fro, self.to))
+
+def rearrange(current_permutation, final_permutation):
```

Comment:
I get your point, but I had a hard time deciding whether to make a new class, since the functionality of the class is quite limited. Hmm... But perhaps one shall consider the extendability of the code... Then having a class is probably a good idea! ^^